app/assets/stylesheets/variables/typography.scss

```
@@ -1,5 +1,5 @@
 $font-serif: 'Droid Serif', Georgia, Times, serif !default;
-$font-sans: 'Dejavu Sans', Helvetica, sans-serif !default;
+$font-sans: 'Atkinson Hyperlegible','Times New Roman', sans-serif !default;
```

```suggestion
$font-sans: 'Atkinson Hyperlegible','Dejavu Sans', Helvetica, sans-serif !default;
```

Could we keep the same font fallback options for non-Latin characters?